<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="video_source">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>999</width>
      <height>999</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>VideoLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Source</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBarrera1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>20</width>
      <height>240</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderBarrera2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>20</width>
      <height>240</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBarrera1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBarrera2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>400</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="counter">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>counter</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleBarrera1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Barrera inferior</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleBarrera2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Barrera superior</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
